### PR TITLE
Add tutorial on how to build an NFT Marketplace

### PR DESCRIPTION
**Pull request by ankg**
Adding a new tutorial that teaches web3 developers how to build an NFT Marketplace from scratch. The tutorial walks them through creating the frontend and backend as well as setting up IPFS to store NFT images.

## Description

Adding a new folder in src/content/developers/tutorials/ called how-to-build-an-nft-marketplace with the below contents
- A markdown file which is the main articles
- Images that are used in the article

## Related Issue

This is not an issue, instead a new article being submitted

### CODE REVIEW

**Comment by ankg** [comment]
Hi @minimalsm - Can you please help review this one?

This is an updated PR based on your response to my previous PR: https://github.com/ethereum/ethereum-org-website/pull/6822

File: src/content/developers/tutorials/how-to-build-an-nft-marketplace/index.md [hunk 1]
```diff
@@ -0,0 +1,1263 @@
...
+Such scale is only achieved with great smart contracts and scalable infrastructure. So if you're a web3 dev looking to improve your web3 development skills, follow along with this tutorial on building an NFT marketplace using [Alchemy](https://www.alchemy.com/), IPFS, Hardhat and ethers.js.
+
+Some things to keep in mind:
+
+- The focus of this tutorial will be on building the smart contract and not building the frontend. However, the [frontend code for an NFT marketplace](https://github.com/OMGWINNING/NFT-Marketplace-Tutorial) is available on GitHub.
+- No backend or database is involved in this tutorial. A backend and database will only be needed when you start to archive data and integrate registration or login features.
+
+### Step 0: Sign up for an Alchemy account and create a new app
+
+If you haven't already, [sign up for your free Alchemy account](https://dashboard.alchemyapi.io/signup/?a=dabb74c129).
```
Comment (Pandapip1) [inline src/content/developers/tutorials/how-to-build-an-nft-marketplace/index.md:39]: Affiliate links need to be disclosed prominently at the beginning of articles.

**Comment by ankg** [comment]
Hi @Pandapip1 - I have added the line at the top. Any other fixes to be done?

**Comment by ankg** [comment]
Ah okay thanks. @minimalsm could you take a look?

**Comment by minimalsm** [comment]
Hey @ankg, thanks for the PR. As we're prepping for The Merge PRs like this are low priority/on the backlog until we finish up all the requirements of that. We'll try to get to it as soon as we're able after The Merge.

A few things on first glance that will get this over the line quicker when we get to review it:

1. We require unique header ids on all markdown headings. Feel free to check out our markdown pages for more examples.

```
Instead of this:
## Introduction

Do this:
## Introduction {#introduction}
```

2. You need to rename you images with actual names and a consistent naming convention, preferably with dashes between each word. Names like `screenshot4023423` won't be accepted and we wouldn't merge this until that is fixed.

3. We cannot accept any PRs that contain tracking links as it violates our privacy policy.

4. Please make the post 'merge-proof'. It should refer to The Merge in the past tense (but it's preferable if it doesn't refer to The Merge at all unless it needs to). I noticed this in the testnet section, but any instances will need fixed.

5. The heading level should start at H2 (H1 is the page title); currently this starts at H3. All headings probably need to be moved up one level.

6. You can check out [our style guide](https://ethereum.org/en/contributing/style-guide/) to see if there is anything else you might think is applicable.

**Comment by corwintines** [comment]
Marking this as abandoned as there haven't been any changes since the last request. @ankg feel free to reopen after addressing @minimalsm last comment and we will take another look.

**Comment by corwintines** [comment]
Also, please fill out an [issue](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=feature+%3Asparkles%3A%2Ccontent+%3Afountain_pen%3A&template=suggest_tutorial.yaml&title=Suggest+a+tutorial) for this so that we can review easier. Preferably you would fill this out before submitting a complete tutorial.

**Pull request closed without merging**